crates/bevy_gltf/src/loader.rs

```diff
@@ -459,9 +463,9 @@ fn load_node(
 
     // create camera node
     if let Some(camera) = gltf_node.camera() {
-        node.insert(VisibleEntities {
-            ..Default::default()
-        });
+        // node.insert(VisibleEntities {
+        //     ..Default::default()
+        // });
```

Comment:
I missed this when considering visibility stuff. Hmm...

Reply:
There were a number of gltf-camera-related issues. Just fixed them!